### ISSUE TITLE
Add a `gg_stop_recording`

### ISSUE THREAD

**Issue by TylerGrantSmith** [thebioengineer/camcorder#10]
Shouldn't there be a method to turn off the recording feature?  I guess I had assumed calling `gg_playback()` would have done it, but that does not appear to be the case.

**Comment by thebioengineer** [comment]
`gg_playback()` does turn off the recording feature. However, a `gg_stop_recording()` does make sense!

**Comment by thebioengineer** [comment]
could you share a screenshot of what was going on?

**Comment by TylerGrantSmith** [comment]
No apparently I cannot.  ImageMagick was erroring when trying to write a file.  I can't seem to replicate it anymore and the procedure seems to work now, and yes `gg_playback()` does stop it.

**Comment by TylerGrantSmith** [comment]
I think maybe one of the `png` that had been produced was corrupted?

```
gg_playback()
Error in magick_image_readpath(enc2native(path), density, depth, strip,  : 
  rsession.exe: ImproperImageHeader `C:\Users\e014307\AppData\Local\Temp\1\RtmpAL3qnt\6b78fbf3938148a28a7e4edb53fb24db.png' @ error/png.c/ReadPNGImage/4050
```

**Comment by thebioengineer** [comment]
that's what it looks like. Which is strange, and I'm not sure how we would be able to fix that. But that would probably be why it didnt stop the recording - the function error-ed out before it hit the part where it turned off recording.

**Comment by thebioengineer** [comment]
addressed in #9
